### DISCUSSION TITLE
Maintained info project

### DISCUSSION THREAD

**Discussion by malgor** [sveltejs/kit#6250] [Ideas]
The latest breaking changes on routing and layouts have taken me a long time to absorb (I still am - my project has been in hiatus for the last week).  I don't think I'm the only one - #6072.  It would be very helpful to have a maintained project that shows the flow of data between the various `handle` and `load` functions on server and client.

By maintained, I mean that it should be available as an option in `npm create svelte@latest` or the rendered output of such a project included in the [documents page](https://kit.svelte.dev/docs/introduction).

**Comment by ZetiMente** [comment]
Ben's Real World Svelte is probably what you're looking for. 

https://github.com/sveltejs/realworld

**Reply by malgor** [reply]
[RealWorld](https://github.com/sveltejs/realworld) is not exactly what I mean.  I need to see how data propagates from server to client and from scripts to components and from layout to page.  What data can the various `locals`, `data` and functions (`load` and `parent`) read and write.  Also, my proposal is that the Info project be maintained by the SvelteKit team so that whenever SvelteKit is updated Info is updated in sync (just as the migration tools and documents are).

**Comment by malgor** [comment]
To resolve this issue I have created such a project.  Is there any interest from the people in charge of SvelteKit in building it into an option in `npm create svelte@latest`?
